package.json

```diff
@@ -46,6 +43,7 @@
     "@babel/core": "^7.12.9",
     "@babel/preset-env": "^7.12.7",
     "@babel/preset-typescript": "^7.12.7",
+    "@types/amqplib": "^0.5.16",
```

Comment:
Is it relevant to this PR?

Reply:
not really to this one, you are right.

Reply:
But on the other hand amqplib has already been added in the previous one so I'll leave it for now I guess.